### PR TITLE
Edits to PGD PR5293

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/pgd/5/durability/commit-scope-rules.mdx [hunk 1]
```diff
@@ -2,49 +2,49 @@
 title: Commit scope rules
 ---
 
-Commit scope rules are at the core of the commit scope mechanism. They define what the commit scope will enforce. 
+Commit scope rules are at the core of the commit scope mechanism. They define what the commit scope enforces. 
 
-Commit scope rules are composed of one or more operations combined with an `AND`. 
+Commit scope rules are composed of one or more operations connected by an `AND`.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/commit-scope-rules.mdx:8]: This seemed more idiomatic and clear to me

File: product_docs/docs/pgd/5/durability/commit-scope-rules.mdx [hunk 2]
```diff
@@ -2,49 +2,49 @@
...
 ```
-commit_scope_group [ confimation_level ] commit_scope_kind
+commit_scope_group [ confirmation_level ] commit_scope_kind
 ```
 
-A full formal syntax diagram is available in the [commit scope reference section.](/pgd/latest/reference/commit-scopes/#commit-scope-syntax).
+A full formal syntax diagram is available in the [Commit scopes](/pgd/latest/reference/commit-scopes/#commit-scope-syntax) reference.
 
-If we look at a typical commit scope rule, we can now break it down into its components:
+This typical commit scope rule can be broken down into its components:
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/commit-scope-rules.mdx:17]: These edits...I was trying to follow the logic. The use of the word "here" was confusing me, and I ultimately decided it all meant to refer to the same example. Let me know if I get any of what was meant here wrong and we can revisit.

File: product_docs/docs/pgd/5/durability/commit-scope-rules.mdx [hunk 3]
```diff
@@ -2,49 +2,49 @@
...
-- `ANY n NOT` &mdash; is followed by an integer value, "n". It translates to any "n" nodes that are not in the listed groups nodes.
-- `ALL NOT` &mdash; is followed by the groups and translates to all nodes that are not in the listed group's nodes.
-- `MAJORITY NOT` &mdash; is followed by the groups and translates to requiring a half, plus one, of the nodes that are not in the listed groups nodes to confirm to give a majority.
+- `ANY n` is followed by an integer value, `n`. It translates to any `n` nodes in the listed group's nodes.
+- `ALL` is followed by the groups and translates to all nodes in the listed groups nodes.
+- `MAJORITY` is followed by the groups and translates to requiring a half, plus one, of the listed group's nodes to confirm, to give a majority.
+- `ANY n NOT` is followed by an integer value, `n`. It translates to any `n` nodes that aren't in the listed group's nodes.
+- `ALL NOT` is followed by the groups and translates to all nodes that aren't in the listed group's nodes.
+- `MAJORITY NOT` is followed by the groups and translates to requiring a half, plus one, of the nodes that aren't in the listed group's nodes to confirm, to give a majority.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/commit-scope-rules.mdx:45]: The em dash treatment usually wouldn't have a lower-cased verb following. These read more like sentences than what I'd call our typical "definition list" that includes the em-dash. I think they read fine without the em-dash (easier to read).

File: product_docs/docs/pgd/5/durability/commit-scope-rules.mdx [hunk 4]
```diff
@@ -2,49 +2,49 @@
...
+The last part of this operation is the commit scope kind, which in this example is `GROUP COMMIT`. `GROUP COMMIT` is a synchronous two-phase commit that's confirmed when any two nodes in the named group confirm they've flushed the transactions changes and made them visible.
 
 ## The commit scope group
 
-There are three kinds of commit scope group, `ANY`, `ALL` and `MAJORITY`. They are all followed by a parenthesized list of one or more groups, which combine to make a pool of nodes that this operation will apply to. This list can be preceded by `NOT` which inverts to pool to all other groups apart from those in the list. Witness nodes are not eligible to be included in this pool as they do not replicate data.
+There are three kinds of commit scope groups: `ANY`, `ALL`, and `MAJORITY`. They're all followed by a list of one or more groups in parentheses. This list of groups combines to make a pool of nodes this this operation applies to. This list can be preceded by `NOT`, which inverts to pool to all other groups apart from those in the list. Witness nodes aren't eligible to be included in this pool, as they don't replicate data.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/commit-scope-rules.mdx:37]: Does the newly formed sentence make sense?

File: product_docs/docs/pgd/5/durability/commit-scope-rules.mdx [hunk 5]
```diff
@@ -53,52 +53,54 @@ PGD nodes can send confirmations for a transaction at different times. In increa
...
 
-For further details see [`LAG CONTROL`](lag-control)
+For more details, see [`LAG CONTROL`](lag-control).
 
 ### `SYNCHRONOUS_COMMIT`
 
-Synchronous Commit is a commit scope option which is designed to be like the legacy `synchronous_commit` option, but accessible within the commit scope environment. Unlike `GROUP COMMIT` it is a synchronous non-two-phase commit operation, with no parameters. The preceding commit scope group controls what groups and confirmation requirements the `SYNCHRONOUS_COMMIT` will use.
+Synchronous Commit is a commit scope option that's designed to be like the legacy `synchronous_commit` option, but it's accessible in the commit scope environment. Unlike `GROUP COMMIT`, it's a synchronous non-two-phase commit operation, and it has no parameters. The preceding commit scope group controls the groups and confirmation requirements the `SYNCHRONOUS_COMMIT` uses.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/commit-scope-rules.mdx:94]: I wasn't sure what you meant by "the preceding commit scope group."

File: product_docs/docs/pgd/5/durability/commit-scope-rules.mdx [hunk 6]
```diff
@@ -53,52 +53,54 @@ PGD nodes can send confirmations for a transaction at different times. In increa
...
 
-For further details see [`SYNCHRONOUS_COMMIT`](synchronous_commit)
+For more details, see [`SYNCHRONOUS_COMMIT`](synchronous_commit).
 
 ## Combining rules
 
-A rule can have multiple operations, combined with an `AND` to form a single rule. For example,
+A rule can have multiple operations connected by an `AND` to form a single rule. For example:
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/commit-scope-rules.mdx:100]: Same as the other edit; seemed more idiomatic.

File: product_docs/docs/pgd/5/durability/group-commit.mdx [hunk 7]
```diff
@@ -124,46 +124,44 @@ does eventually COMMIT even though the client might receive an abort message.
...
 copied to all remaining nodes through normal logical replication. 
 
-The origin node seeks confirmations from other nodes, as per rules in the group
-commit grammar. If it gets confirmations from minimum required nodes in the
-cluster, it decides to commit this transaction moving onto the second phase (commit) 
-where it also sends this decision via replication to other nodes which will also eventually commit on getting this message.
+The origin node seeks confirmations from other nodes, as per rules in the Group
+Commit grammar. If it gets confirmations from the minimum required nodes in the
+cluster, it decides to commit this transaction moving onto the second phase (commit). 
+In the commit phase, it also sends this decision by way of replication to other nodes. Those nodes will also eventually commit on getting this message.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/group-commit.mdx:140]: Please check these edits, as it was a little hard to follow.

File: product_docs/docs/pgd/5/durability/group-commit.mdx [hunk 8]
```diff
@@ -124,46 +124,44 @@ does eventually COMMIT even though the client might receive an abort message.
...
+There's a possibility of failure at various stages. For example, the origin
 node may crash after preparing the transaction. Or the origin and one or more
-replicas may crash. 
+replicas may crash.
 
 This leaves the prepared transactions in the system. The `pg_prepared_xacts`
 view in Postgres can show prepared transactions on a system. The prepared
-transactions could be holding locks and other resources and they therefore need
-to be either aborted or committed. That decision has to be made with a consensus
+transactions might be holding locks and other resources and they therefore need
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/group-commit.mdx:148]: Wasn't sure if it should be "can," "might," or maybe it should be recast as: It's possible for the prepared transactions to be holding locks and other resources. They therefore...

Now that I'm looking at this again, I think that's a better edit and that a separate sentence isn't a bad idea. What think?

File: product_docs/docs/pgd/5/reference/routing.mdx [hunk 9]
```diff
@@ -38,14 +38,14 @@ bdr.alter_proxy_option(proxy_name text, config_key text, config_value text);
...
+The proxy options that you can change using this function are:
+- `listen_address` &mdash; Address for the proxy to listen on. Default is `{0.0.0.0}`.
+- `listen_port` &mdash; Port for the proxy to listen on. Default is `6432`.
+- `max_client_conn` &mdash; Maximum number of connections for the proxy to accept. Default is `32767`.
+- `max_server_conn` &mdash; Maximum number of connections the proxy can make to the Postgres node. Default is `32767`.
+- `server_conn_timeout` &mdash; Connection timeout for server connections, in seconds. Default is `2`.
+- `server_conn_keepalive` &mdash; Keepalive interval for server connections, in seconds. Default is `10`.
+- `consensus_grace_period` &mdash; Duration for which proxy continues to route even upon loss of a Raft leader, in seconds. If set to 0s, proxy stops routing immediately. Default is generally `6` for local proxies and `12` for global proxies. These values are overridden if `raft_response_timeout`, `raft_global_election_timeout`, or `raft_group_election_timeout` are set to values other than their defaults.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/routing.mdx:48]: The "in seconds" edit...is that what was meant? It seemed more apt to qualify it in the description rather than in the defaults, if that's correct.

File: product_docs/docs/pgd/5/consistency/eager.mdx [hunk 10]
```diff
@@ -54,9 +54,9 @@ In case of an origin node failure, the remaining nodes eventually (after at leas
 
 With single-node Postgres, or even with PGD in its default asynchronous
 replication mode, errors at `COMMIT` time are rare. The added synchronization 
-step due to the use of a commit scope using eager
+step due to the use of a commit scope using Eager
```
Comment (djw-m) [inline product_docs/docs/pgd/5/consistency/eager.mdx:57]: eager is lower case in the commit scope.

File: product_docs/docs/pgd/5/durability/commit-scope-rules.mdx (hunk 2 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/commit-scope-rules.mdx:17]:
```suggestion
A typical commit scope rule, such as `ANY 2 (group) GROUP COMMIT`, can be broken down into its components:
```

Maybe clearer - just lose the code block below.

Oh and the "here" is something I completely forgot as this content started as a conversational examination of scopes.

File: product_docs/docs/pgd/5/durability/synchronous_commit.mdx [hunk 11]
```diff
@@ -22,22 +24,19 @@ SELECT bdr.add_commit_scope(
...
+ `received`              | A remote PGD node confirms the transaction once it's been fully received and is in the in-memory write queue.
  `replicated`            | Same behavior as `received`.
  `durable`               | Confirms the transaction after all of its changes are flushed to disk. Analogous to `synchronous_commit = on` in legacy synchronous replication.
  `visible` (default)     | Confirms the transaction after all of its changes are flushed to disk and it's visible to concurrent transactions. Analogous to `synchronous_commit = remote_apply` in legacy synchronous replication.
 
 ## Details
 
-Currently `SYNCHRONOUS_COMMIT` does not use the confirmation levels of the commit scope rule syntax.
-
+Currently `SYNCHRONOUS_COMMIT` doesn't use the confirmation levels of the commit scope rule syntax.
```
Comment (djw-m) [inline product_docs/docs/pgd/5/durability/synchronous_commit.mdx:42]: Oh this may have already been changed in the docs anyway. Will need to check.

**Review by djw-m — COMMENTED** [review]
Comments as requested.

File: product_docs/docs/pgd/5/durability/commit-scope-rules.mdx (hunk 1 above)
Comment (ebgitelman) [reply product_docs/docs/pgd/5/durability/commit-scope-rules.mdx:8]: Or maybe just be more explicit? 

Commit scope rules are composed of one or more operations that work in combination. Use an `AND` between rules.

Does that work?

File: product_docs/docs/pgd/5/durability/commit-scope-rules.mdx (hunk 4 above)
Comment (ebgitelman) [reply product_docs/docs/pgd/5/durability/commit-scope-rules.mdx:31]: I think this construction is a little easier to parse. Okay?

File: product_docs/docs/pgd/5/durability/commit-scope-rules.mdx [hunk 12]
```diff
@@ -53,52 +47,54 @@ PGD nodes can send confirmations for a transaction at different times. In increa
...
 
-For further details see [`LAG CONTROL`](lag-control)
+For more details, see [`LAG CONTROL`](lag-control).
 
 ### `SYNCHRONOUS_COMMIT`
 
-Synchronous Commit is a commit scope option which is designed to be like the legacy `synchronous_commit` option, but accessible within the commit scope environment. Unlike `GROUP COMMIT` it is a synchronous non-two-phase commit operation, with no parameters. The preceding commit scope group controls what groups and confirmation requirements the `SYNCHRONOUS_COMMIT` will use.
+Synchronous Commit is a commit scope option that's designed to be like the legacy `synchronous_commit` option, but it's accessible in the commit scope environment. Unlike `GROUP COMMIT`, it's a synchronous non-two-phase commit operation, and it has no parameters. The preceding commit scope group controls the groups and confirmation requirements the `SYNCHRONOUS_COMMIT` uses.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/commit-scope-rules.mdx:88]:
```suggestion
Synchronous Commit is a commit scope option that's designed to be like the legacy `synchronous_commit` option, but it's accessible in the commit scope environment. Unlike `GROUP COMMIT`, it's a synchronous non-two-phase commit operation, and it has no parameters. The commit commit scope group that comes before this option controls the groups and confirmation requirements the `SYNCHRONOUS_COMMIT` uses.
```

File: product_docs/docs/pgd/5/durability/commit-scope-rules.mdx (hunk 6 above)
Comment (ebgitelman) [reply product_docs/docs/pgd/5/durability/commit-scope-rules.mdx:94]: See earlier comment.

File: product_docs/docs/pgd/5/durability/group-commit.mdx (hunk 7 above)
Comment (ebgitelman) [reply product_docs/docs/pgd/5/durability/group-commit.mdx:140]: I guess because of its Latin roots...I think this is one I discussed with Dee Dee and we decided not to use it. It's a pretty typical style guide thing, although I don't think we lose any clarity. It's like e.g. and i.e....perfectly understandable, just a style guide choice.

File: product_docs/docs/pgd/5/durability/group-commit.mdx (hunk 8 above)
Comment (ebgitelman) [reply product_docs/docs/pgd/5/durability/group-commit.mdx:148]: I think that's a bit clearer in the active voice.

File: product_docs/docs/pgd/5/reference/nodes-management-interfaces.mdx [hunk 13]
```diff
@@ -235,11 +235,11 @@ bdr.alter_node_option(node_name text,
 -   `config_value` &mdash; New value to be set for the given key.
 
 The node options that can be changed using this function are:
-- `route_priority` &mdash; Relative routing priority of the node against other nodes in the same node group. Default is '-1'.
-- `route_fence` &mdash; Whether the node is fenced from routing; when true, the node can't receive connections from PGD Proxy. Default is 'f' (false).
-- `route_writes` &mdash; Whether writes can be routed to this node, that is, whether the node can become write leader. Default is 't' (true) for data nodes and 'f' (false) for other node types.
-- `route_reads` &mdash; Whether read-only connections can be routed to this node. Currently reserved for future use. Default is 't' (true) for data and subscriber-only nodes, 'f' (false) for witness and standby nodes.
-- `route_dsn` &mdash; The dsn that the proxy will use to connect to this node. This is optional; if not set it defaults to the node's node_dsn value.
+- `route_priority` &mdash; Relative routing priority of the node against other nodes in the same node group. Default is `-1`.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/nodes-management-interfaces.mdx:238]:
```suggestion
- `route_priority` &mdash; Relative routing priority of the node against other nodes in the same node group. Default is `'-1'`.
```
Okay, I've sometimes seen the single quotes used when it was meant to be back ticks, so yes, using both makes it clearer (to me at least!) that the single quotes are part of the value. I'll fix the rest of these.

File: product_docs/docs/pgd/5/reference/nodes-management-interfaces.mdx [hunk 14]
```diff
@@ -235,11 +235,11 @@ bdr.alter_node_option(node_name text,
...
 
 The node options that can be changed using this function are:
-- `route_priority` &mdash; Relative routing priority of the node against other nodes in the same node group. Default is '-1'.
-- `route_fence` &mdash; Whether the node is fenced from routing; when true, the node can't receive connections from PGD Proxy. Default is 'f' (false).
-- `route_writes` &mdash; Whether writes can be routed to this node, that is, whether the node can become write leader. Default is 't' (true) for data nodes and 'f' (false) for other node types.
-- `route_reads` &mdash; Whether read-only connections can be routed to this node. Currently reserved for future use. Default is 't' (true) for data and subscriber-only nodes, 'f' (false) for witness and standby nodes.
-- `route_dsn` &mdash; The dsn that the proxy will use to connect to this node. This is optional; if not set it defaults to the node's node_dsn value.
+- `route_priority` &mdash; Relative routing priority of the node against other nodes in the same node group. Default is `'-1'`.
+- `route_fence` &mdash; Whether the node is fenced from routing. When true, the node can't receive connections from PGD Proxy. Default is `f` (false).
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/nodes-management-interfaces.mdx:239]:
```suggestion
- `route_fence` &mdash; Whether the node is fenced from routing. When true, the node can't receive connections from PGD Proxy. Default is `'f'` (false).
```

File: product_docs/docs/pgd/5/reference/nodes-management-interfaces.mdx [hunk 15]
```diff
@@ -235,11 +235,11 @@ bdr.alter_node_option(node_name text,
...
+- `route_writes` &mdash; Whether writes can be routed to this node, that is, whether the node can become write leader. Default is `t` (true) for data nodes and `f` (false) for other node types.
+- `route_reads` &mdash; Whether read-only connections can be routed to this node. Currently reserved for future use. Default is `t` (true) for data and subscriber-only nodes, `f` (false) for witness and standby nodes.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/nodes-management-interfaces.mdx:241]:
```suggestion
- `route_reads` &mdash; Whether read-only connections can be routed to this node. Currently reserved for future use. Default is `'t'` (true) for data and subscriber-only nodes, `'f'` (false) for witness and standby nodes.
```

File: product_docs/docs/pgd/5/consistency/eager.mdx (hunk 10 above)
Comment (ebgitelman) [reply product_docs/docs/pgd/5/consistency/eager.mdx:57]:
```suggestion
step due to the use of a commit scope using eager
```
Should eager be tagged? Is it an option?

File: product_docs/docs/pgd/5/durability/commit-scope-rules.mdx (hunk 1 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/commit-scope-rules.mdx:8]: Go with that then.

File: product_docs/docs/pgd/5/consistency/eager.mdx (hunk 10 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/consistency/eager.mdx:57]:
```suggestion
step due to the use of a commit scope using `eager`
```

File: product_docs/docs/pgd/5/durability/synchronous_commit.mdx (hunk 11 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/synchronous_commit.mdx:42]: We're good.

File: product_docs/docs/pgd/5/durability/commit-scope-rules.mdx [hunk 16]
```diff
@@ -2,49 +2,43 @@
 title: Commit scope rules
 ---
 
-Commit scope rules are at the core of the commit scope mechanism. They define what the commit scope will enforce. 
+Commit scope rules are at the core of the commit scope mechanism. They define what the commit scope enforces. 
 
-Commit scope rules are composed of one or more operations combined with an `AND`. 
+Commit scope rules are composed of one or more operations connected by an `AND`.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/commit-scope-rules.mdx:7]:
```suggestion
Commit scope rules are composed of one or more operations that work in combination. Use an AND between rules.
```

File: product_docs/docs/pgd/5/durability/commit-scope-rules.mdx [hunk 17]
```diff
@@ -53,52 +47,54 @@ PGD nodes can send confirmations for a transaction at different times. In increa
...
-Synchronous Commit is a commit scope option which is designed to be like the legacy `synchronous_commit` option, but accessible within the commit scope environment. Unlike `GROUP COMMIT` it is a synchronous non-two-phase commit operation, with no parameters. The preceding commit scope group controls what groups and confirmation requirements the `SYNCHRONOUS_COMMIT` will use.
+Synchronous Commit is a commit scope option that's designed to be like the legacy `synchronous_commit` option, but it's accessible in the commit scope environment. Unlike `GROUP COMMIT`, it's a synchronous non-two-phase commit operation, and it has no parameters. The commit commit scope group that comes before this option controls the groups and confirmation requirements the `SYNCHRONOUS_COMMIT` uses.
 
-For further details see [`SYNCHRONOUS_COMMIT`](synchronous_commit)
+For more details, see [`SYNCHRONOUS_COMMIT`](synchronous_commit).
 
 ## Combining rules
 
-A rule can have multiple operations, combined with an `AND` to form a single rule. For example,
+A rule can have multiple operations connected by an `AND` to form a single rule. For example:
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/commit-scope-rules.mdx:94]:
```suggestion
Commit scope rules are composed of one or more operations that work in combination. Use an AND to form a single rule. For example:
```

**Comment by ebgitelman** [comment]
@djw-m This was hanging around. It should be good to go. Thanks!

**Review by djw-m — APPROVED** [review]
LGTM
